madara-prover-rpc-server/src/hints.rs

```diff
@@ -126,7 +146,9 @@ pub fn hint_processor() -> BuiltinHintProcessor {
         LOAD_BOOTLOADER_CONFIG.to_string(),
         unimplemented_hint.clone(),
     );
-    hint_processor.add_hint(SAVE_OUTPUT_POINTER.to_string(), unimplemented_hint.clone());
+    hint_processor.add_hint(
+        SAVE_OUTPUT_POINTER.to_string(),
+        Rc::new(HintFunc(Box::new(save_output_pointer_hint.clone()))));
```

Comment:
Also, clippy is not happy with this `.clone()`.

Reply:
ee21c8c